testdata/cheats/Cheats.sol

```diff
@@ -2,6 +2,15 @@
 pragma solidity >=0.8.0;
 
 interface Cheats {
+    // Possible caller modes for readCallers()
+    enum CallerMode {
+        None,
+        Broadcast,
+        RecurrentBroadcast,
+        Prank,
+        RecurrentPrank
+    }
```

Comment:
this we'd need to move to forge-std?

Reply:
yea @xeno097 once this is merged do you mind opening a PR to forge-std to add these to [`VmSafe`](https://github.com/foundry-rs/forge-std/blob/b00f504daf0bdd8cf2e67973e2c86bd213cda400/src/Vm.sol#L12)

Reply:
Yep no problem! I think I also need to add a new page to the fpundry book with documentation for the cheatcode right?

Reply:
Yes please that would be great! 🙏